docs/docusaurus.config.js

```diff
@@ -0,0 +1,137 @@
+// @ts-check
+// `@type` JSDoc annotations allow editor autocompletion and type checking
+// (when paired with `@ts-check`).
+// There are various equivalent ways to declare your Docusaurus config.
+// See: https://docusaurus.io/docs/api/docusaurus-config
+
+import { themes as prismThemes } from "prism-react-renderer";
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
```

Comment:
Config should be customized to fit the Joyboy, can you change necessary fields?

Reply:
There are still some fields that must changed in the config.
- copyright
- url (https://docs.joyboycommunity for now)
- tagline
- organizationName should be `keep-starknet-strange`
- docs & blog editUrl
- logo alt